CMakeLists.txt

```diff
@@ -73,17 +73,27 @@ set(EVENT_VERSION_MINOR ${EVENT_GIT___VERSION_MINOR})
 set(EVENT_VERSION_PATCH ${EVENT_GIT___VERSION_PATCH})
 set(EVENT_VERSION_STAGE ${EVENT_GIT___VERSION_STAGE})
 
-
-set(EVENT_ABI_MAJOR ${EVENT_VERSION_MAJOR})
-set(EVENT_ABI_MINOR ${EVENT_VERSION_MINOR})
-set(EVENT_ABI_PATCH ${EVENT_VERSION_PATCH})
+if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
```

Comment:
why linux only? what about other unix like OSes?

Reply:
There are too many Unix-like OSes, such as Linux, Darwin, Android, SunOS, AIX, FreeBSD, CygWin and etc.  Unfortunately, I don't have as many machines to test. Can you tell me exactly which systems should I test?  Thank you.

Reply:
Let's just make it non-win32?

Reply:
@ygj6 can you fix this bit and rebase/squash the patchset (also it will be great if you will include some details of this change into commit description) so I can apply it?